src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7110.mdx

```diff
@@ -0,0 +1,119 @@
+
+
+* Custom Event Limit Increase  [1036](https://github.com/newrelic/newrelic-java-agent/pull/1036)
+
+This version increases the default limit of custom events from 10,000 events per minute to 30,000 events per minute. In the scenario that custom events were being limited, this change will allow more custom events to be sent to New Relic. There is also a new configurable maximum limit of 100,000 events per minute. To change the limits, see the documentation for [max_samples_stored](/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file/#Custom_Events). To learn more about the change and how to determine if custom events are being dropped, see our Explorers Hub [post](https://discuss.newrelic.com/t/send-more-custom-events-with-the-latest-apm-agents/190497).
+
+
+* Code-level metrics on by default [1037](https://github.com/newrelic/newrelic-java-agent/pull/1037)
+
+The code-level metrics functionality introduced with agent 7.10 is now enabled by default. This feature will cause an increase in the consumption of data. The amount depends on the application. To disable code-level metrics, see instructions on our [code-level metrics documentation](/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file#code-level-metrics).
+
+
+## Fixes
+* Prevent an exception from being thrown in the Jedis instrumentation [1011](https://github.com/newrelic/newrelic-java-agent/pull/1011)
+* Improvement on Http4s transactions [1006](https://github.com/newrelic/newrelic-java-agent/pull/1006)
+* Fix noticeError API not verifying whether errors were expected [1014](https://github.com/newrelic/newrelic-java-agent/pull/1014)
+* Adds command for Lettuce queries to clusters [1031](https://github.com/newrelic/newrelic-java-agent/pull/1031)
+* Fix exception when building up the agent jar from a clean repo [1048](https://github.com/newrelic/newrelic-java-agent/pull/1048)
+* Better error handling for code-level metrics [1021](https://github.com/newrelic/newrelic-java-agent/pull/1021) [1051](https://github.com/newrelic/newrelic-java-agent/pull/1051)
+* Fix HttpUrlConnection spans not terminating on exception [1053](https://github.com/newrelic/newrelic-java-agent/pull/1053)
+
+
+## Deprecation notice
+The following instrumentation modules are being deprecated for removal:
+* cassandra-datastax-2.1.2
+* httpclient-3.0
+* jdbc-embedded-derby-10.10.1.1
+* jetty-7
+* jetty-7.6
+* jetty-9
+* jetty-9.0.4
+* jetty-9.1
+* mongodb-2.12
+* mongodb-2.14
+* mongodb-3.0
+* okhttp-3.0.0
+* okhttp-3.4.0
+* okhttp-3.5.0
+
+The http.responseCode, response.status and response.statusMessage transaction/span attributes are deprecated and will be removed in a future release. These have been replaced by http.statusCode and http.statusText.
```

Comment:
@meiao how about adding some backticks for keywords:

The `http.responseCode`, `response.status` and `response.statusMessage` transaction/span attributes are deprecated and will be removed in a future release. These have been replaced by `http.statusCode` and `http.statusText`.